adapters/huaweiads/huaweiadstest/exemplary/nativeSingleImage.json

```diff
@@ -166,6 +164,7 @@
                     "appId": "101219405",
                     "appPromotionChannel": "401721412",
                     "clickUrl": "https://ads.huawei.com/usermgtportal/home/index.html#/",
+                    "cta": "%e5%ae%89%e8%a3%85",
```

Comment:
As per code change,  `cta` value should be assigned as value to `desc` label . But value for `desc` is empty. 

Should update tests so that `cta` value is reflected in `adm`

https://github.com/prebid/prebid-server/blob/9a8d83326ee4cd173b5ae552f6649f371b88aedc/adapters/huaweiads/huaweiadstest/exemplary/nativeSingleImage.json#L302

Reply:
thanks for the comment. i changed and tested it again. Can you check it please? @onkarvhanumante